components/scream/cmake/Scorpio.cmake

```diff
@@ -0,0 +1,110 @@
+# Use Macros.cmake to get info on netcdf paths.
+# Note: the inputs are supposed to be *the name* of the variables storing the result
+# Note: Keep this a FUNCTION, not a MACRO, to avoit polluting the calling scope
+#       with all the stuff from Macros.cmake
+function (GetNetcdfLibs)
+  # Sanity check
+  if (NOT CIME_BUILD)
```

Comment:
Note to self: if we move scorpio support outside `mct_coupling`, to get it also in standalone scream runs, we should upgrade this script to do configure/build scorpio.

Also, we might think about moving this script inside ekat's cmake goodies.

Reply:
yes, I was thinking we should move to the share folder sooner than later.  We could add that to this PR, or a quick and neat one after this goes in.

Reply:
There's no rush. Just a thing to keep in mind.